diff --git a/node_modules/react-native-nitro-modules/android/CMakeLists.txt b/node_modules/react-native-nitro-modules/android/CMakeLists.txt
index 1234567..abcdefg 100644
--- a/node_modules/react-native-nitro-modules/android/CMakeLists.txt
+++ b/node_modules/react-native-nitro-modules/android/CMakeLists.txt
@@ -70,6 +70,7 @@
     target_link_libraries(
         NitroModules
         ReactAndroid::reactnative                 # <-- RN: Native Modules umbrella prefab
+        c++_shared
     )
 else()
     target_link_libraries(
